### DISCUSSION TITLE
不知道作者能否接受付费定制

### DISCUSSION THREAD

**Discussion by Arm0ne** [SPlayer-Dev/SPlayer#136] [想法 Ideas]
我作为一个音乐爱好者但代码小白，一直想做一个自己的音乐网站，作者这个项目从颜值到功能我觉得已经符合我心中大部分的想法了，不知道能否沟通一下，付费请作者开发一个版本。

**Comment by imsyy** [comment]
是还需要什么功能呢 😲

**Reply by Arm0ne** [reply]
不知道你方不方便私聊？fod1619@gmail.com
